osbs/cli/main.py

```diff
@@ -600,12 +603,19 @@ def cli():
                               help="perform a scratch build")
     build_parser.add_argument("--yum-proxy", action='store', required=False,
                               help="set yum proxy to repos from koji/add-yum-repo params")
-    build_parser.add_argument("--worker", action="store_true", required=False,
-                              default=False, help="create worker build")
     build_parser.add_argument("--platform", action='store', required=False,
-                              help="platform name to use")
+                              help="platform name to use (worker build)")
+    build_parser.add_argument("--platforms", action='append', metavar='PLATFORM',
+                              help="name of each platform to use (orchestrator build)")
```

Comment:
Is there any validation to prevent `--worker --platforms ...` or `--orchestrator --platform ...`?